packages/palette/src/elements/DonutChart/DonutChart.tsx

```diff
@@ -147,12 +151,18 @@ export const DonutChart: React.FC<DonutChartProps> = ({
     </svg>
   )
 
+  const ChartSVG = () => (
+    <>
+      {tooltip}
+      {labels}
+      {svg}
+    </>
+  )
+
```

Comment:
This is leading to some strangeness in regards to `hasEnteredViewport`. Would be worth playing with it a bit.

Reply:
Why did you change this? You're creating a new react component type on every render here, which makes React unable to track changes properly.

Reply:
This was just an experiment to see the animation appear on load, nothing definitive. @sepans is on it.